languagetool-language-modules/ga/src/main/java/org/languagetool/tagging/ga/Retaggable.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 Jim O'Regan <jaoregan@tcd.ie>
```

Comment:
Is this a different license (not LGPL) on purpose? Not that it's a problem, but keeping the licenses coherent would be nice.

Reply:
Yes, the licence here (and in Utils.java) is on purpose: these are pieces that are quite general purpose in terms of processing Irish, and don't depend on anything else, so I'd like that anyone who wants to process Irish can use them without having to ask me.